server/routes/api/prospects.js

```diff
@@ -112,4 +121,42 @@ router.delete("/:id", (req, res) => {
   
 });
 
+// @route POST /api/prospects/upload
+// @desc Upload Prospects with a csv file
+// @access Authenticated Users
+router.post("/upload", upload.single('file'), (req, res) => {
+  const csvData = [];
+  let count = {};
+  const file = req.file;
+
+  const { errors, isValid } = validateFile(file);
+  if(!isValid) {
+      return res.status(400).send(errors);
+  }
+
+  fs.createReadStream(file.path)
+      .pipe(csv())
+      .on('data', (row) => {
+        csvData.push(row);
+      })
+      .on("end", async function () {
```

Comment:
You can use the arrow function syntax with async functions too. There are some inconsistencies here where sometimes you use the `function` keyword in a callback while other times you use an arrow function.